api/v2/api.go

```diff
@@ -204,11 +208,40 @@ func (api *API) getStatusHandler(params general_ops.GetStatusParams) middleware.
 
 func (api *API) getReceiversHandler(params receiver_ops.GetReceiversParams) middleware.Responder {
 	api.mtx.RLock()
-	defer api.mtx.RUnlock()
+	configReceivers := api.receivers
+	api.mtx.RUnlock()
+
+	receivers := make([]*open_api_models.Receiver, 0, len(configReceivers))
+	for _, r := range configReceivers {
+		integrations := make([]*open_api_models.Integration, 0, len(r.Integrations()))
+
+		for _, integration := range r.Integrations() {
+			notify, duration, err := integration.GetReport()
+			iname := integration.String()
+			sr := integration.SendResolved()
+
+			integrations = append(integrations, &open_api_models.Integration{
+				Name:               &iname,
+				SendResolve:        &sr,
+				LastNotify:         notify.UTC().String(),
```

Comment:
_As it's still a draft ignore it if you already planned to do so_: can we make sure that the date is in ISO 8601 format?